toolkit/imageconfigs/marketplace-gen2-fips.json

```diff
@@ -0,0 +1,83 @@
+{
+    "Disks": [
+        {
+            "PartitionTableType": "gpt",
+            "MaxSize": 5000,
+            "Artifacts": [
+                {
+                    "Name": "cblmariner-gen2-fips",
+                    "Type": "vhd"
```

Comment:
Should this not be `vhdx` ?
Or is this one more `quirk` we need to be aware of

Reply:
This is an Azure-specific quirk, where the actual image type that Azure recognizes is `vhd`, even though the actual image has UEFI Gen2 VM properties